roboteam_ai/src/control/positionControllers/BallHandlePosControl.cpp

```diff
@@ -79,7 +79,7 @@ RobotCommand BallHandlePosControl::getRobotCommand(const RobotPtr &r,
 
```

Comment:
Why make such a mess of all the print statements here? I liked the way it was before better because atleast there is a clear place where they are. Now you removed some of the switch cases but also kept some. If you change it, change them all.

Reply:
no i did not remove them and i did not make a mess. I made them testable, meaning that now they have a proper input and output